hack/build-coreos-manifest.go

```diff
@@ -20,15 +19,23 @@ import (
 const (
 	streamRHCOSJSON = "data/data/coreos/rhcos.json"
 	streamFCOSJSON  = "data/data/coreos/fcos.json"
+	streamSCOSJSON  = "data/data/coreos/scos.json"
 	fcosTAG         = "okd"
+	scosTAG         = "scos"
```

Comment:
```suggestion
        okdTag          = "okd"
	fcosTAG         = "fcos"
	scosTAG         = "scos"
```
the `okd` and `fcos` Go build tags used elsewhere in the installer are synonymous. Let's add an `okdTAG` here and treat it like `fcosTAG`.